### PR TITLE
feat: halt thorchain operations during hardfork

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

⚠️ MERGE ME at 5PM UTC on 09/04 i.e two hours before the hard fork, not before

Does what it says on the box - to be reverted once https://github.com/shapeshift/web/pull/7510 goes in 

## Issue (if applicable)

N/A

## Risk

> High Risk PRs Require 2 approvals

None, temporary halt
 
> What protocols, transaction types, wallets or contract interactions might be affected by this PR?

## Testing

- THORChain swapper, LP, Lending, savers and RUNEPool are disabled 

### Operations

- [ ] :checkered_flag: My feature is behind a flag and doesn't require operations testing (yet)

## Screenshots (if applicable)

Screenshot 2024-09-03 at 17 43 17
Screenshot 2024-09-03 at 17 42 59
Screenshot 2024-09-03 at 17 41 52
Screenshot 2024-09-03 at 17 33 06

### CODE REVIEW

File: .env.base [hunk 1]
```diff
@@ -176,4 +176,4 @@ REACT_APP_SENTRY_DSN_URL=https://c612e7f4ef0637e4add433a2f4683aa8@o4507174990905
 REACT_APP_RFOX_PROXY_CONTRACT_ADDRESS=0xac2a4fd70bcd8bab0662960455c363735f0e2b56
 
 # Zerion
-REACT_APP_ZERION_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zerion
\ No newline at end of file
+REACT_APP_ZERION_BASE_URL=https://api.proxy.shapeshift.com/api/v1/zerion
```
Comment (gomesalexandre) [inline .env.base:179]: EOL shenanigans

**Comment by 0xean** [comment]
to be merged tomorrow 17 UTC.  2 hours ahead of TC hardfork

**Review by 0xApotheosis — COMMENTED** [review]
Sanity checked `yarn env app` config:

✅ No THORChain swapper available

Screenshot 2024-09-04 at 6 18 13 AM

✅ Savers deposit and withdraws disabled

Screenshot 2024-09-04 at 6 18 47 AM

❌ Lending & Pools routes are gone. This is not ideal, and too course of a flag - users should still be able to view their positions.

Screenshot 2024-09-04 at 6 23 05 AM

**Comment by gomesalexandre** [comment]
> ❌ Lending & Pools routes are gone. This is not ideal, and too course of a flag - users should still be able to view their positions.

Nice catch @0xApotheosis! Reverted the top-level flags and only kept the granular ones in a1cb69df75

image
image

**Review by 0xApotheosis — APPROVED** [review]
Much better 🙏 

Positions are accessible, but can't be interacted with. Exactly what we want!

Screenshot 2024-09-04 at 6 53 15 AM

**Comment by kaladinlight** [comment]
hotfix release: https://github.com/shapeshift/web/pull/7690

**Pull request closed without merging**